x-pack/test/security_solution_api_integration/test_suites/lists_and_exception_lists/lists_items/trial_license_complete_tier/items/find_list_items.ts

```diff
@@ -44,9 +37,9 @@ export default ({ getService }: FtrProviderContext): void => {
           .send()
           .expect(400);
 
-        expect(body).to.eql({
+        expect(body).toEqual({
           error: 'Bad Request',
-          message: '[request query]: Invalid value "undefined" supplied to "list_id"',
+          message: expect.any(String),
```

Comment:
In this particular case the error message is even shorter `[request query]: list_id: Required`. I put it here as expected value. But generally speaking such tests are brittle. For example I had to fix much more cases like this one in https://github.com/elastic/kibana/issues/183661 while test names don't mention an error message verification.

I think we can use exact messages in simple cases and `expect.toMatch()` and `expect.any()` in cases when brittleness has a high chance.

Reply:
Yeah for this case and the one below at https://github.com/elastic/kibana/pull/185865/files#diff-a8197c324b1e134ff57c060cc25a2529d9e05357424893c753ee48d5aaa04929R55 I think it's nice to have the expected string included explicitly. I wouldn't expect the strings to be changing often, really they should only change when we do large scale changes like this where we're swapping out the whole validation schema framework.

The more brittle strings imo are the Elasticsearch info/error messages, since those can change without us knowing when Elasticsearch is updated. But here we should have full control of the schema validation messages and they shouldn't change and break these tests unexpectedly.